src/SFA.DAS.ApplyService.InternalApi/Infrastructure/TokenService.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using SFA.DAS.ApplyService.Configuration;
+
+namespace SFA.DAS.ApplyService.InternalApi.Infrastructure
+{
+    public class TokenService : ITokenService
```

Comment:
Has this been copied from somewhere?  It looks like something that might already exist or could be reusable?